build/image-gallery.js

```diff
@@ -1218,6 +1220,7 @@ ImageGallery.defaultProps = {
   swipingTransitionDuration: 0,
   slideInterval: 3000,
   swipeThreshold: 30,
+  numberOfImagesToPreLoad: 0,
```

Comment:
possible to get test around the new prop?

Reply:
This lib currently has no unit tests at all.

Is adding jest test support required to approve?

Reply:
We agreed to write up a tech debt story to add tests.